tests/test_handlers.py

```diff
@@ -7,18 +7,21 @@
 - The expected output in the __output__ folder.
 """
 
+import inspect
 import subprocess
 from pathlib import Path
 
 import pytest
 
+from unblob.handlers import _ALL_MODULES_BY_PRIORITY
 from unblob.processing import DEFAULT_DEPTH, process_file
 
 TEST_DATA_PATH = Path(__file__).parent / "integration"
 TEST_INPUT_DIRS = list(TEST_DATA_PATH.glob("**/__input__"))
 TEST_CASE_DIRS = [p.parent for p in TEST_INPUT_DIRS]
 TEST_OUTPUT_DIRS = [p / "__output__" for p in TEST_CASE_DIRS]
 TEST_IDS = [p.name for p in TEST_CASE_DIRS]
+HANDLER_MODULE_PATH = Path(__file__).parent.parent / "unblob" / "handlers"
```

Comment:
A more future-proof and less-verbose way would be:
```suggestion
from unblob import handlers
HANDLERS_PACKAGE_PATH = Path(handlers.__file__).parent
```
Also it's not a module, it's a package.